frontend/src/lib/components/left-bar/LeftBar.svelte

```diff
@@ -123,211 +124,233 @@
         return mdiHelp;
     }
   }
+
+  const queueLockoutPopupId = 'mod-details-compat-ea';
+  const queueLockoutPopup = {
+    event: 'hover',
+    target: queueLockoutPopupId,
+    middleware: {
+      offset: 4,
+    },
+    placement: 'bottom-end',
+  } satisfies PopupSettings;
 </script>
 
 <div class="flex flex-col h-full p-4 space-y-4 h-md:space-y-8 left-bar w-[22rem] min-w-[22rem] ">
   <div class="flex flex-col flex-auto h-full w-full space-y-4 h-md:space-y-8 overflow-y-auto">
-    <div class="flex flex-col gap-2">
-      <span class="pl-4 sticky top-0 z-[1] bg-surface-50-900-token">
-        <T defaultValue="Select Game Installation" keyName="left-bar.game-version"/>
-      </span>
-      <Select
-        name="installsCombobox"
-        class="w-full h-8"
-        buttonClass="bg-surface-200-700-token px-4 text-sm"
-        disabled={!$canChangeInstall}
-        itemActiveClass="!bg-surface-300/20"
-        itemClass="bg-surface-50-900-token"
-        items={_.orderBy($installs)}
-        value={$selectedInstall ?? ''}
-        on:change={installSelectChanged}
-      >
-        <svelte:fragment slot="item" let:item>
-          <span class="flex items-center min-w-0">
-            {#if $installsMetadata[item]?.state === ficsitcli.InstallState.VALID}
-              <SvgIcon class="!w-5 !h-5 mr-2 shrink-0" icon={iconForInstallType($installsMetadata[item].info?.type)}/>
-              <span class="truncate min-w-0">
-                {$installsMetadata[item]?.info?.launcher}
-              </span>
-              <span class="shrink-0 ml-1">({#if $installsMetadata[item]?.info?.location === common.LocationType.LOCAL}{$installsMetadata[item]?.info?.branch}{:else}CL{$installsMetadata[item]?.info?.version}{/if})</span>
-            {:else if $installsMetadata[item]?.state === ficsitcli.InstallState.LOADING}
-              <T defaultValue="Loading..." keyName="left-bar.install-loading"/>
-            {:else if $installsMetadata[item]?.state === ficsitcli.InstallState.INVALID}
-              <T defaultValue="Invalid" keyName="left-bar.install-invalid"/>
-            {:else}
-              <T defaultValue="Unknown" keyName="left-bar.install-unknown"/>
-            {/if}
-          </span>
-        </svelte:fragment>
-        <svelte:fragment slot="itemTrail" let:item>
-          <Tooltip popupId={installOptionPopupId(item)}>
-            <div class="flex flex-col">
-              <span>{item}</span>
+    <div class="h-md:space-y-8" use:popup={queueLockoutPopup}>
+
```

Comment:
I removed it because it seems to have no effect on the final product.
Without: 
![image](https://github.com/user-attachments/assets/4b5ce7ee-6eb3-4337-9ce9-ec63692f5a7b)
With: 
![image](https://github.com/user-attachments/assets/79940416-db60-45f8-bf13-8a33f6d88a10)

Reply:
That's because of your window size (what `h-md` means). When you make the window smaller that class no longer applies to there's no spacing anymore